setup.go

```diff
@@ -61,6 +62,9 @@ type Config struct {
 
 	// URL with schema supported by zap. Use zap.RegisterSink
 	URL string
+
+	// Labels is a set of key-values to apply to all loggers
+	Labels []string
```

Comment:
This should be a map (or a slice of `Label{Key, Value string}` structs). The format we use in environment variables shouldn't leak into the go API.